doc/system_requirements.md

```diff
@@ -36,48 +40,82 @@ The following list gives you an overview of terms and abbreviations commonly use
 
 Features are the highest level requirements in this document that describe the main functionality of EM.
 
-### List Extensions
+### Managing Extensions
+
+EM allows managing extensions.
+
+#### List Extensions
 `feat~list-extensions~1`
 
 EM lists extensions.
 
 Needs: dsn
 
-### Install Extensions
+#### Install Extensions
 `feat~install-extension~1`
 
 EM allows the DBA to install extensions.
 
 Needs: req
 
-### Configure an Extension
+#### Configure an Extension
 `feat~configure-extension~1`
 
 EM allows the DBA to configure an extension, e.g. in order to set up a Virtual Schema source system.
 
 Needs: req
 
-### Updating an Extension
+#### Updating an Extension
 `feat~update-extension~1`
 
 EM allows DBA to install a new version of an extension that was already installed in an older version.
 
 Needs: req
 
-### Uninstall an Extension
+#### Uninstall an Extension
 `feat~uninstall-extension~1`
 
 EM allows the DBA to uninstall an extension.
 
 Needs: req
 
+### Developing Extensions
+`feat~developing-extensions~1`
+
+As extensions are an important part of EM's ecosystem it must be easy for developers to create and maintain new extensions.
+
+Needs: req
+
+### REST Interface
+`feat~rest-interface~1`
+
+EM provides a REST interface for clients.
+
+Needs: req
+
 <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 ## Functional Requirements
 
-### Extensions
+### Managing Extensions
+
+#### EM Finds Available Extensions
+`req~finding-available-extensions~1`
+
+EM automatically finds available extensions at runtime. It allows adding new extensions at runtime.
```

Comment:
I guess this hints at having no local cache of the remote extension list or at least a regular cache-freshness check. Good point.

Reply:
Correct. We don't have a cache at the moment. We can add one later if necessary.